src/chat.ts

```diff
@@ -170,7 +170,17 @@ export class ChatProvider implements vscode.WebviewViewProvider {
         });
       } else {
         // Reveal the directory in the explorer
-        vscode.commands.executeCommand("revealInExplorer", link);
+        vscode.commands
+          .executeCommand("revealInExplorer", filePath)
+          .then(() =>
+            // This is a little hack.
+            // 
+            // There's some issue with the revealInExplorer command which means when the panel changes 
+            // (e.g. from sourcery to the explorer) the instruction to actually focus the file path
+            // seems to get dropped. By calling it again (after the first command succeeds) we can 
+            // make sure the file actually gets navigated to.
+            vscode.commands.executeCommand("revealInExplorer", filePath)
```

Comment:
interesting - can probably raise this as an issue with the vscode team as well

Reply:
Yeah I was actually put on to this as from this issue: https://github.com/microsoft/vscode/issues/160504#issuecomment-1247660237

Idk how much work it might be to set up an example though